old/industry.csv

```diff
@@ -1,4 +1,5 @@
 name,affiliation,homepage,scholarid,company
+T. Niranjan,Stony Brook University,https://www.linkedin.com/in/thirumale-niranjan-8b41391,NOSCHOLARPAGE,Epsilon
```

Comment:
AFAICT this faculty member was not in CSrankings, so no need to delete them by creating this entry. Please clarify.

Reply:
I see. I'll close the PR then